pkg/grapicmd/internal/module/generator/template/init.go

```diff
@@ -6,52 +6,87 @@ import (
+var _Initd135936e91856b6159ac2eedcf89aa9f07773f82 = "package main\n\nimport (\n\t\"context\"\n\n\t\"github.com/izumin5210/grapi/pkg/grapiserver\"\n)\n\nfunc run() error {\n\t// Application context\n\tctx := context.Background()\n\n\ts := grapiserver.New(\n\t\tgrapiserver.WithDefaultLogger(),\n\t\tgrapiserver.WithServers(\n\t\t// TODO\n\t\t),\n\t)\n\treturn s.ServeContext(ctx)\n}\n"
+var _Init8d21956ba8abe388f964e47be0f7e5d170a2fce5 = ""
 var _Init71ed560e812a4261bc8b56d9feaef4800830e0b7 = ""
-var _Initd135936e91856b6159ac2eedcf89aa9f07773f82 = "package main\n\nimport (\n\t\"github.com/izumin5210/grapi/pkg/grapiserver\"\n)\n\nfunc run() error {\n\ts := grapiserver.New(\n\t\tgrapiserver.WithDefaultLogger(),\n\t\tgrapiserver.WithServers(\n\t\t// TODO\n\t\t),\n\t)\n\treturn s.Serve()\n}\n"
-var _Init881d845139e03b8e1e2dafd175e793d03f9bacaf = "// +build tools\n\npackage tools\n\n// tool dependencies\nimport (\n\t_ \"github.com/golang/protobuf/protoc-gen-go\"\n\t_ \"github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway\"\n\t_ \"github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger\"\n\t_ \"github.com/izumin5210/grapi/cmd/grapi\"\n\t_ \"github.com/izumin5210/grapi/cmd/grapi-gen-command\"\n\t_ \"github.com/izumin5210/grapi/cmd/grapi-gen-scaffold-service\"\n\t_ \"github.com/izumin5210/grapi/cmd/grapi-gen-service\"\n\t_ \"github.com/izumin5210/grapi/cmd/grapi-gen-type\"\n)\n"
 
 // Init returns go-assets FileSystem
-var Init = assets.NewFileSystem(map[string][]string{"/app/server": []string{".keep.tmpl"}, "/api/protos/type": []string{".keep.tmpl"}, "/api/protos": []string{".keep.tmpl"}, "/": []string{".gitignore.tmpl", "tools.go", "grapi.toml.tmpl"}, "/cmd": []string{}, "/cmd/server": []string{"main.go.tmpl", "run.go.tmpl"}, "/app": []string{}, "/api": []string{}}, map[string]*assets.File{
-	"/cmd/server/run.go.tmpl": &assets.File{
-		Path:     "/cmd/server/run.go.tmpl",
-		FileMode: 0x1a4,
-		Mtime:    time.Unix(1553312167, 1553312167203472863),
-		Data:     []byte(_Initd135936e91856b6159ac2eedcf89aa9f07773f82),
-	}, "/tools.go": &assets.File{
-		Path:     "/tools.go",
-		FileMode: 0x1a4,
-		Mtime:    time.Unix(1553227614, 1553227614146442611),
-		Data:     []byte(_Init881d845139e03b8e1e2dafd175e793d03f9bacaf),
-	}, "/api/protos/type/.keep.tmpl": &assets.File{
-		Path:     "/api/protos/type/.keep.tmpl",
+var Init = assets.NewFileSystem(map[string][]string{"/": []string{".gitignore.tmpl", "tools.go", "grapi.toml.tmpl"}, "/api": []string{}, "/api/protos": []string{".keep.tmpl"}, "/api/protos/type": []string{".keep.tmpl"}, "/app": []string{}, "/app/server": []string{".keep.tmpl"}, "/cmd": []string{}, "/cmd/server": []string{"main.go.tmpl", "run.go.tmpl"}}, map[string]*assets.File{
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
File is not `gofmt`-ed with `-s` (gofmt)